src/azure-cli/azure/cli/command_modules/resource/tests/latest/recordings/test_aaz_resource_id_arg_fmt.yaml

```diff
@@ -14,7 +14,7 @@ interactions:
       - AZURECLI/2.45.0 azsdk-python-azure-mgmt-resource/22.0.0 Python/3.10.9 (Linux-5.15.0-1031-azure-x86_64-with-glibc2.31)
         VSTS_7b238909-6802-4b65-b90d-184bca47f458_build_220_0
     method: GET
-    uri: https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/test_resource_id_arg_fmt000001?api-version=2022-09-01
+    uri: https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/test_resource_id_arg_fmt000001?api-version=2024-11-01
```

Comment:
Please try to re-record these YAML files using live testing instead of just replacing the api-version in the YAML files

Reply:
I'm having trouble finding any account where I can run `azdev test resource --live` without running into permissions issues.

Can I leave most of these files untouched (e.g., still using the 2022-09-01 API version)? The API update is expected to have a minimal effect on only a handful of operations.

Reply:
Thanks for the guidance! All changes in this PR were now authored by that pipeline.

Reply:
I did need to manually patch a large number of recordings after running the pipeline, but this PR should be ready for review now.